packages/twenty-front/src/modules/object-record/object-options-dropdown/components/ObjectOptionsDropdownRecordGroupFieldsContent.tsx

```diff
@@ -36,6 +37,7 @@ export const ObjectOptionsDropdownRecordGroupFieldsContent = () => {
     recordIndexId,
     objectMetadataItem,
     onContentChange,
+    resetContent,
```

Comment:
I would use an event instead or directly call a useResetContent hook here to avoid props drilling function, which is an anti pattern.

Reply:
What do you mean by props drilling function ? `resetContent` is coming from the `ObjectOptionsDropdownContext`

Reply:
Ah yes sorry !